domain/keymanager/state/state.go

```diff
@@ -143,7 +143,7 @@ WHERE s.algorithm = $userPublicKeyInsert.algorithm
 SELECT &userPublicKeyId.id
 FROM user_public_ssh_key
 WHERE user_uuid = $userPublicKeyInsert.user_uuid
-AND public_key = $userPublicKeyInsert.public_key
+AND fingerprint = $userPublicKeyInsert.fingerprint
```

Comment:
This is the fix for the unique constraint issue I ran into.

Reply:
Cheers and good find.